src/amigaDebug.ts

```diff
@@ -458,24 +465,21 @@ export class AmigaDebugSession extends LoggingDebugSession {
 			case '1.8m':
 				config.set('slow_memory', '1792');
 				break;
-			default:
-				config.delete('slow_memory');
 			}
 		}
 
-		try {
-			fs.writeFileSync(defaultPath, stringifyCfg(config));
-		} catch(e) {
-			this.sendErrorResponse(response, 103, `Unable to write emulator config ${defaultPath}.`);
-			return;
-		}
-
-		// all WinUAE options now in config file
 		const emuPath = isWin
 			? path.join(binPath, "winuae-gdb.exe")
 			: path.join(binPath, "fs-uae", "fs-uae");
 
-		const emuArgs = isWin ? [ '-portable' ] : [ defaultPath ];
+		const emuArgs = [
+			...(isWin
+				// all WinUAE options now in config file
+				? [ '-portable' ]
+				// FS-UAE options as args
+				: [...config].map(([k, v]) => `--${k}=${v}`)),
+			...args.emuargs
```

Comment:
This line causes `TypeError: args.emuargs is not iterable` when no emuargs are specified, thus breaking debugging for everyone in 1.7.0